test/datasets/cmuarctic_test.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from torchaudio.datasets import cmuarctic
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+    normalize_wav,
+)
+
+
+class TestCMUARCTIC(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    URL = "aew"  # default url in CMUARCTIC
+    samples = []
+
+    @classmethod
+    def setUpClass(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        sample_rate = 16000
+        seed = 42
+        utterance = "This is a test utterance."
+
+        base_dir = os.path.join(cls.root_dir, "ARCTIC", "cmu_us_" + cls.URL + "_arctic")
+        # Contains utterance ID & sentence prompts
```

Comment:
This kind of comments, ones not adding more information than what code expresses are not necessary. You can remove them.